### PR TITLE
apply setup-py-upgrade

### PR DESCRIPTION
**Pull request by graingert**
[pipx](https://github.com/pipxproject/pipx) run [setup-py-upgrade](https://github.com/asottile/setup-py-upgrade) .

### CODE REVIEW

**Comment by daltonmatos** [comment]
Hello @graingert, since a new version was released (which has the old setup.py) now it's conflicting.

Could you please update you branch? The only difference is the `version` variable, it should now be `0.5.1`.

Thanks.

**Comment by graingert** [comment]
Ah I just ran `pipx run setup-py-upgrade .` and fixed the lint errors

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+url = https://github.com/b2wdigital/aiologger
+project_urls =
+    Documentation=https://aiologger.readthedocs.io/en/latest/
+    Code=https://github.com/b2wdigital/aiologger
+    Issue tracker=https://github.com/b2wdigital/aiologger/issues
+author = Diogo Magalhães Martins
+author_email = magalhaesmartins@icloud.com
+keywords = logging json log output
+classifiers =
+    Development Status :: 4 - Beta
```
Comment (diogommartins) [inline setup.cfg:17]: I think that atm we're way past Beta. How do you feel about changing this classifier, @daltonmatos  ?

Comment (daltonmatos) [reply setup.cfg:17]: Agree! We can remove this beta status. 🎉

Comment (diogommartins) [reply setup.cfg:17]:
```suggestion
    Development Status :: 5 - Production/Stable
```

Comment (graingert) [reply setup.cfg:17]: @daltonmatos @diogommartins I'd be tempted to wait until the loop_compat code was removed before marking v1/stable

Comment (diogommartins) [reply setup.cfg:17]: Makes sense. Let leave it for another moment, then!

**Review by diogommartins — APPROVED** [review]

**Pull request merged**